client/state/desktop/middleware.js

```diff
@@ -24,15 +24,10 @@ const debug = debugFactory( 'desktop:middleware' );
 export const desktopMiddleware = () => {
 	return ( next ) => ( action ) => {
 		switch ( action.type ) {
-			case NOTIFICATIONS_UNSEEN_COUNT_SET: {
+			case NOTIFICATIONS_PANEL_TOGGLE: {
```

Comment:
This will result in the IPC event firing twice - once when it's toggled open, and once when it's closed, which may cause some unintended side effects (for example if they leave their notifications panel open, step away, receive a bunch of notifications, return to their computer and then close the panel).

Might be a little more robust if we combined this with the selector `isNotificationsOpen` from `calypso/state/selectors/is-notifications-open';`?